getting_started/setup.adoc

```diff
@@ -9,53 +9,86 @@
-
-The following sections describe the system requirements per component type.
+== Overview
+ifdef::openshift-origin[]
+OpenShift components can be installed across multiple hosts. The following sections outline the system requirements and instructions for preparing your environment and hosts before installing OpenShift.
+endif::[]
 
-*Masters*
+ifdef::openshift-enterprise[]
+OpenShift components can be installed across multiple hosts. During the Beta 1 phase, we recommend installing a  link:../architecture/kubernetes_infrastructure.html#master[master] on one host, and two link:../architecture/kubernetes_infrastructure.html#node[nodes] on two separate hosts.
+endif::[]
 
-Master hosts must meet the following system requirements:
+== System Requirements
+The system requirements vary per host type:
 
-- Physical or virtual system
+[cols="1,7"]
+|===
+|Masters a|- Physical or virtual system
 ifdef::openshift-origin[]
-- Base OS: Fedora 21, CentOS 7, or Red Hat Enterprise Linux Server 7.0
+- Base OS: Fedora 21, CentOS 7, or RHEL 7.1 Beta ("Minimal" installation option)
 endif::[]
 ifdef::openshift-enterprise[]
-- Base OS: Red Hat Enterprise Linux Server 7.0
+- Base OS: RHEL 7.1 Beta ("Minimal" installation option)
 endif::[]
 - 2 vCPU
 - Minimum 8 GB RAM
 - Minimum 30 GB hard disk space
 
-*Nodes*
-
-Node hosts must meet the following system requirements:
-
-- Physical or virtual system, or an instance running on a public IaaS
+| Nodes a| - Physical or virtual system, or an instance running on a public IaaS
 ifdef::openshift-origin[]
-- Base OS: Fedora 21, CentOS 7, or Red Hat Enterprise Linux Server 7.0
+- Base OS: Fedora 21, CentOS 7, or RHEL 7.1 Beta ("Minimal" installation option)
 endif::[]
 ifdef::openshift-enterprise[]
-- Base OS: Red Hat Enterprise Linux Server 7.0
+- Base OS: RHEL 7.1 Beta ("Minimal" installation option)
 endif::[]
 - 1 vCPU
 - Minimum 8 GB RAM
 - Minimum 15 GB hard disk space
+|===
+
+*Security Warning*
+
+OpenShift runs Docker containers on your system, and in some cases, such as build operations and the registry service, it does so using privileged containers. Furthermore, those containers access your host's Docker daemon and perform `docker build` and `docker push` operations. As such, you should be aware of the inherent security risks associated with performing `docker run` operations on arbitrary images as they effectively have root access. This is particularly relevant when running the OpenShift nodes directly on your host system.
+
+For more information, see these articles:
+
+- http://opensource.com/business/14/7/docker-security-selinux
+- https://docs.docker.com/articles/security/
+
+The OpenShift security model will continue to evolve and tighten as we head towards production-ready code.
```

Comment:
Doesn't translate well to this format.  "OpenShift will increase the security constraints on containers in later beta releases."